### ISSUE TITLE
Move to upstream dex

### ISSUE THREAD

**Issue by pivotal-jwinters** [concourse/concourse#3306]
## What challenge are you facing?

Maintaining a fork of dex

## What would make this better?

Get all our changes merged into upstream

**Comment by pivotal-jwinters** [comment]
currently waiting on https://github.com/dexidp/dex/pull/1370

**Comment by vito** [comment]
Consolidating this into https://github.com/concourse/concourse/issues/4983
